datasets/inspecties/dataset.json

```diff
@@ -0,0 +1,281 @@
+        "identifier": "id",
+        "required": [
+          "id",
+          "schema"
+        ],
+        "display": "id",
+        "properties": {
+          "schema": {
+            "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+          },
+          "id": {
+            "type": "string",
+            "description": "Unieke aanduiding van het record."
+          },
+          "environment": {
+            "type": "string",
+            "description": "Lijst bevat de volgende eigenschappen. environment Omgevingsparameter waarin de inspectie heeft plaatsgevonden (bijv. amsterdam, amsterdam2019, amsterdamcentrum, etc.)"
+          },
+          "inspectionRoundType": {
+            "type": "string",
+            "description": " "
+          },
+          "inspectionRoundId": {
+            "type": "string",
+            "description": "Het id van de schouwronde"
+          },
+          "inspectionRoundUri": {
+            "type": "string",
+            "description": "Link naar de gehele schouwronde"
+          },
+          "inspectionRoundName": {
+            "type": "string",
+            "description": "Naam van de schouwronde"
+          },
+          "inspectieRoundPeriodeStart": {
+            "type": "string",
+            "description": "Unix timestamp met geplande startdatum en tijd"
+          },
+          "inspectionRoundPeriodeEnd": {
+            "type": "string",
+            "description": "Unix timestamp met geplande einddatum en tijd"
+          },
+          "inspectionRoundModifiededAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum en tijd"
+          },
+          "inspectionIds": {
+            "type": "string",
+            "description": "Het id van de inspectie"
+          },
+          "inspectionCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum"
+          },
+          "resultsAboveMinvalue": {
+            "type": "boolean",
+            "provenance": "all_results_above_minimum_value",
+            "description": "true ( = geen afkeur) / false ( = bevat afkeur)"
+          },
+          "inspectionFinishedAt": {
+            "type": "string",
+            "description": "Unix timestamp met sluitingsdatum en tijd"
+          },
+          "inspectionResultFiledAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum"
+          },
+          "closingUserName": {
+            "type": "string",
+            "description": "Gebruiker die de inspectie heeft gefinaliseerd"
+          },
+          "scoreId": {
+            "type": "string",
+            "description": "Het id van het resultaat"
+          },
+          "score": {
+            "type": "string",
+            "enum": [
+              "A+",
+              "A",
+              "B",
+              "C",
+              "D",
+              "NVT"
```

Comment:
Is er een reden om een expliciete "NVT" te willen hebben ipv. gewoon het veld weglaten in de rijen waar het niet is ingevuld? Rijen met ontbrekende velden kunnen gevonden worden met `score[isnull]=true`.

Reply:
die NVT staat er letterlijk in de data. Het is dus een onderdeel van de lijst.